txstratum/api.py

```diff
@@ -112,21 +111,10 @@ async def submit_job(self, request: web.Request) -> web.Response:
             self.log.debug('non-standard-tx', data=data)
             return web.json_response({'error': 'non-standard-tx'}, status=400)
 
-        # Check for banned inputs.
-        for txin in tx.inputs:
-            if txin.tx_id in self.banned_tx_ids:
-                self.log.info('txin-banned', data=data)
+        for tx_filter in self.tx_filters:
+            if await tx_filter.check_tx(tx, data):
                 return web.json_response({'error': 'invalid-tx'}, status=400)
```

Comment:
Should we let the filter decides that return?

Reply:
I don't think so, we could later have more complex filters so returning a named tuple (e.g. `(block, message)`) could be better but for now the simple boolean response indicating if the tx should be blocked is a better format.

Reply:
Ok. I agree. Thanks!